org.gtk.Gtk3theme.Jade-1.json

```diff
@@ -0,0 +1,46 @@
+{
+   "id":"org.gtk.Gtk3theme.Jade-1",
+   "branch":"3.22",
+   "runtime":"org.freedesktop.Sdk",
+   "build-extension":true,
+   "sdk":"org.freedesktop.Sdk",
+   "runtime-version":"19.08",
+   "appstream-compose":false,
+   "separate-locales":false,
+   "modules":[
+      {
+         "name":"Jade-1",
+         "buildsystem":"simple",
+         "build-commands":[
+            "tar -xf jade-1-theme.tar.xz",
+            "install -Dm644 Jade-1/gtk-3.22/gtk.css /usr/share/runtime/share/themes/Jade-1/gtk-3.0",
+            "install -Dm644 Jade-1/gtk-3.0/colours.css /usr/share/runtime/share/themes/Jade-1/gtk-3.0",
+            "install -Dm644 --target-directory=/usr/share/runtime/share/themes/Jade-1/gtk-3.0/assets Jade-1/gtk-3.0/assets/*",
+            "install -Dm644 Jade-1/index.theme /usr/share/runtime/share/themes/Jade-1/index.theme"
+         ],
+         "sources":[
+            {
+               "type":"file",
+               "url":"https://github.com/madmaxms/theme-jade-1/releases/download/v1.8/jade-1-theme.tar.xz",
+               "dest-filename": "jade-1-theme.tar.xz",
+               "sha256":"4f15f4b6a89e3bcaf265ec7c653ebfcd8fdb8ba25b11a95764e05871a0440636"
+            }
+         ]
+      },
+      {
+         "name":"appdata",
+         "buildsystem":"simple",
+         "build-commands":[
+            "mkdir -p ${FLATPAK_DEST}/share/appdata",
+            "cp org.gtk.Gtk3theme.Jade-1.appdata.xml ${FLATPAK_DEST}/share/appdata",
+            "appstream-compose --basename=org.gtk.Gtk3theme.Jade-1 --prefix=${FLATPAK_DEST} --origin=flatpak org.gtk.Gtk3theme.Jade-1"
+         ],
+         "sources":[
+            {
+               "type":"file",
+               "path":"org.gtk.Gtk3theme.Jade-1.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html